lms/pshell.py

```diff
@@ -6,6 +7,12 @@
 
 
 def setup(env):
+    sys.path = ["."] + sys.path
+
+    from tests import factories  # pylint:disable=import-outside-toplevel
+
+    sys.path = sys.path[1:]
```

Comment:
The library `pylti` [contains a `tests` package inside the `pylti` package](https://github.com/mitodl/pylti/tree/master/pylti/tests) and this gets installed at `tox/dev/lib/python3.6/site-packages/tests/`. This means that `import tests` or `from tests import factories` tries to import from pylti's tests dir. This makes it impossible to import our own test factories. So I had to hack the import path